tests/test_infer/test_models/model_utils/baichuan2_7B/generation_utils.py

```diff
@@ -0,0 +1,81 @@
+from queue import Queue
```

Comment:
Same as above. Remove original model and config

Reply:
Done